src/PropertySet.cc

```diff
@@ -39,13 +40,13 @@ namespace base {
 namespace {
 
 /**
- * Append the contents of a vector<T> to a vector<boost::any>
+ * Append the contents of a vector<T> to a vector<std::any>
  *
- * This method exists because vector<boost::any>.insert mis-behaves for vector of bool,
+ * This method exists because vector<std::any>.insert mis-behaves for vector of bool,
  * resulting in a vector with elements that are a strange type.
```

Comment:
It might be worth checking whether this statement is actually true for `std::any`. I would expect the two implementations to have different quirks.

Reply:
This is still the case. Replacing with push_back fails the python unit tests.